internal/api/router.go

```diff
@@ -293,6 +295,15 @@ func NewRouter(handlers Handlers, cfg *config.Configuration, logger *logger.Logg
 			costSheet.GET("/breakdown/:subscription_id", handlers.CostSheet.GetCostBreakDown)
 			costSheet.POST("/roi", handlers.CostSheet.CalculateROI)
 		}
+		// Credit note routes
+		creditNotes := v1Private.Group("/creditnotes")
```

Comment:
I think we have replaced all cost sheet changes
and the `plan.POST("/:id/sync/subscriptions", handlers.Plan.SyncPlanPrices)` api as well while resolving merge conflicts